engine/api/artifact/artifact.go

```diff
@@ -201,8 +201,22 @@ func LoadArtifacts(db gorp.SqlExecutor, pipelineID int64, applicationID int64, e
 		art := sdk.Artifact{}
 		var md5sum, sha512sum, objectpath sql.NullString
 		var size, perm sql.NullInt64
-		err = rows.Scan(&art.ID, &art.Name, &art.Tag, &art.DownloadHash, &size, &perm, &md5sum, &sha512sum, &objectpath, &art.Pipeline, &art.Project, &art.Application, &art.Environment)
-		if err != nil {
+
+		if err := rows.Scan(
```

Comment:
if it only shocks me, keep this syntax but I found this `if` very weird. I prefer 2 lines in this case, if you disagree don't change